conf/cmi/block.block.chatleijuke_2.yml

```diff
@@ -1,32 +1,26 @@
-uuid: 7b6f839b-1b81-4eda-92c5-87e116686a03
+uuid: 716d9214-6b6b-4377-a479-30fc1e5f6e06
 langcode: en
 status: true
 dependencies:
   module:
     - helfi_platform_config
-    - language
     - system
   theme:
     - hdbt_subtheme
-id: hdbt_subtheme_kuurahealthchat
+id: chatleijuke_2
 theme: hdbt_subtheme
 region: attachments
-weight: -10
+weight: 0
 provider: null
-plugin: kuura_health_chat
+plugin: chat_leijuke
 settings:
-  id: kuura_health_chat
-  label: 'Kuura Health Chat'
+  id: chat_leijuke
+  label: 'Chat Leijuke Kuura'
   label_display: '0'
   provider: helfi_platform_config
+  chat_title: 'Kysy terveydestä ja koronasta'
```

Comment:
I assume we need proper English translation for this and FI & SV translations as well.